adapters/pom.xml

```diff
@@ -32,6 +32,7 @@
     <module>mqtt-vertx</module>
     <module>mqtt-vertx-base</module>
     <module>kura</module>
+    <module>hono-adapter-amqp-vertx</module>
```

Comment:
can you name the module in analogy to the other adapter modules, please, i.e. `amqp-vertx`